lib/blocks.php

```diff
@@ -191,6 +194,108 @@ function gutenberg_register_core_block_styles( $block_name ) {
 	}
 }
 
+/**
+ * Change the way block styles get loaded depending on their size.
+ *
+ * Optimizes performance and sustainability of block styles.
+ *
+ * @return void
+ */
+function gutenberg_optimize_block_styles_loading() {
+
+	$total_inline_limit = 20000;
+	/**
+	 * The maximum size of inlined styles in bytes.
+	 *
+	 * @param int $total_inline_limit The file-size threshold, in bytes.
+	 * @return int
+	 */
+	$total_inline_limit = apply_filters( 'styles_inline_size_limit', $total_inline_limit );
+
+	global $wp_styles;
+	$styles = array();
+
+	// Build an array of styles that have a path defined.
+	foreach ( $wp_styles->queue as $handle ) {
+		if ( wp_styles()->get_data( $handle, 'path' ) ) {
```

Comment:
There was a check that ensured that the file exists. We better keep it, no?

Reply:
:laughing: I was just pushing that now